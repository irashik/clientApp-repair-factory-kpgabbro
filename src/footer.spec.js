import React from 'react';

import FooterComponent  from './footer';



       jest.mock('react-router-bootstrap');
       jest.mock('react-bootstrap/Button');
       jest.mock('react-bootstrap/ButtonGroup');
      // jest.mock('react-bootstrap/Nav');
      // jest.mock('react-bootstrap/Navbar');



describe("<FooterComponent />", () => {
    test('should render correctly', () => {

        const component = shallow(<FooterComponent />)
        expect(component).toMatchSnapshot();
    });
});

