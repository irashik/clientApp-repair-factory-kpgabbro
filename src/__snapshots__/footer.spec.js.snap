// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterComponent /> should render correctly 1`] = `
<footer>
  <h5>
    Разработчик: Ирашин Дмитрий
  </h5>
  <h5>
    Licensed 
  </h5>
  <h5>
    Инструкция
  </h5>
  <h5>
    О предприятии
  </h5>
</footer>
`;
